### PR TITLE
initial commit of the X2K appyter

### PR DESCRIPTION
**Pull request by alextsigh**

### CODE REVIEW

**Comment by u8sand** [comment]
@alextsigh it looks like you have a typo in `appyters.json` for the icon.

**Comment by u8sand** [comment]
Please use `appyter nbclean your_notebook.ipynb`

**Comment by u8sand** [comment]
@alextsigh please review the "details" on the validation. In this case, you're missing imports in `requirements.txt`, in this case it's `pandas`

File: appyters/X2K_Appyter/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+git+git://github.com/Maayanlab/appyter.git
+requests
+numpy
```
Comment (ekropiwnicki) [inline appyters/X2K_Appyter/requirements.txt:3]: add pandas to your requirements.txt

**Pull request merged**